### DISCUSSION TITLE
Share store service sometimes throws error in onPrepare hook

### DISCUSSION THREAD

**Discussion by unknowntester404** [webdriverio/webdriverio#13145] [Q&A]
randomly from time to time I receive the following error in before hook
```
file:///*/node_modules/@wdio/shared-store-service/build/client.js:78
    throw new Error(`${err.response?.body || 'Shared store server threw an error'}`);
          ^
Error: Invalid JSON
    at errHandler (file:///*/node_modules/@wdio/shared-store-service/build/client.js:78:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

The set-up is in:

```
    const masterFiles = [];
    for (let i = MAX_INSTANCES; i > 0; i--) {
      masterFiles.push(
        await Files.createFile({
          name: "FileName",
          extension: 'csv'
        })
      );
    }
    await setResourcePool("masterFiles", masterFiles);
```

I saw these: 

- https://github.com/webdriverio/webdriverio/pull/11254
- https://github.com/webdriverio/webdriverio/issues/11208

and was curious if there might be a workaround on fixing this intermitent issue due to server not being initialized.

Thanks

**Comment by christian-bromann** [comment]
> and was curious if there might be a workaround on fixing this intermitent issue due to server not being initialized.

I am not sure if this is connected to the shared store server not being initialized. I don't think we want to advertise workarounds and instead foster collaboration to help us resolve this bug. Any contributions are much appreciated.
